src/server/api/organization-settings.ts

```diff
@@ -41,10 +55,17 @@ const SETTINGS_DEFAULTS: IOrganizationSettings = {
     "I'm opting you out of texts immediately. Have a great day.",
   showContactLastName: false,
   showContactCell: false,
-  confirmationClickForScriptLinks: true
+  confirmationClickForScriptLinks: true,
+  startCampaignRequiresApproval: false
```

Comment:
This isn't a type specified in `IOrganizationSettings`, and it doesn't seem to be used anywhere too.